cloud/doc-chat-2.py

```diff
@@ -0,0 +1,188 @@
+"""
+2-agent doc-chat:
+WriterAgent is in charge of answering user's question.
+Breaks it down into smaller questions (if needed) to send to DocAgent,
+who has access to the docs via a vector-db.
+
+Run like this:
+python3 examples/docqa/doc-chat-2.py
```

Comment:
@chrahul5 I think these instructions need to be changed or removed since this is for cloud run?

Reply:
> @chrahul5 I think these instructions need to be changed or removed since this is for cloud run?

I guess this file should be removed entirely from the cloud folder?